lib/msal-core/tests/MSAL.spec.ts

```diff
@@ -679,49 +669,50 @@ describe('Msal', function (): any {
         expect(decodeURIComponent(result[4])).toContain("https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-implicit-grant-flow?name=value&name2=value2");
     });
 
-    it('tests if you get the state back in tokenReceived callback, if state is a number', function () {
-        spyOn(msal, 'getUserState').and.returnValue("1234");
-        msal._loginInProgress = true;
-        var errDesc = '', token = '', err = '', tokenType = '', state= '' ;
-        var callback = function (valErrDesc:string, valToken:string, valErr:string, valTokenType:string, valState: string) {
-            errDesc = valErrDesc;
-            token = valToken;
-            err = valErr;
-            tokenType = valTokenType;
-            state= valState;
-        };
-
-        msal._tokenReceivedCallback = callback;
-        msal.loginRedirect();
-        expect(errDesc).toBe(ErrorDescription.loginProgressError);
-        expect(err).toBe(ErrorCodes.loginProgressError);
-        expect(token).toBe(null);
-        expect(tokenType).toBe(Constants.idToken);
-        expect(state).toBe('1234');
-        msal._loginInProgress = false;
-    });
-
-    it('tests if you get the state back in tokenReceived callback, if state is a url', function () {
-        spyOn(msal, 'getUserState').and.returnValue("https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-implicit-grant-flow?name=value&name2=value2");
-        msal._loginInProgress = true;
-        var errDesc = '', token = '', err = '', tokenType = '', state= '' ;
-        var callback = function (valErrDesc:string, valToken:string, valErr:string, valTokenType:string, valState: string) {
-            errDesc = valErrDesc;
-            token = valToken;
-            err = valErr;
-            tokenType = valTokenType;
-            state= valState;
-        };
-
-        msal._tokenReceivedCallback = callback;
-        msal.loginRedirect();
-        expect(errDesc).toBe(ErrorDescription.loginProgressError);
-        expect(err).toBe(ErrorCodes.loginProgressError);
-        expect(token).toBe(null);
-        expect(tokenType).toBe(Constants.idToken);
-        expect(state).toBe('https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-implicit-grant-flow?name=value&name2=value2');
-        msal._loginInProgress = false;
-    });
+    // We no longer return state as a part of the error, so these tests need to be updated
```

Comment:
delete comments

Reply:
Had some conversations about returning state as part of the error object, so I will be uncommenting this in a future PR and fixing this functionality